etc/scripts/release.hs

```diff
@@ -264,8 +274,10 @@ rules global@Global{..} args = do
                                 ,"/t", "http://timestamp.verisign.com/scripts/timestamp.dll"
                                 ,out])
                             (removeFile out)
+            -- FIXME testing if the stripping was causing the segfault
             Linux ->
-                cmd "strip -p --strip-unneeded --remove-section=.comment -o"
+                -- cmd "strip -p --strip-unneeded --remove-section=.comment -o"
+                cmd "cp" $ reverse
```

Comment:
Is it possible the reason this causes problems is that you're using the host's `strip` rather than Alpine's `strip`?

Reply:
I just did a test, and when I used Alpine's `strip` it didn't lead to the segfault.  However, the `stack` executable was unchanged after `strip` (same size and SHA sum), so it doesn't seem that it actually did anything (Ubuntu's `strip` also didn't actually shrink the file, although it did modify it and cause the segfault).

This executable is quite a bit bigger than that generated by the Nix process (59MiB vs. 27 MiB).  Not sure it really matters.

Reply:
I'd like something smaller, but I'm not willing to complicate the build process for it. I tried `split-sections`, but it didn't help either. I'll document that this is expected behavior for now.

Question though: do we need the stripping on OS X? `stack copy` should automatically perform stripping, which may be why using Alpine's `strip` causes no improvements.

Reply:
I'm not sure off the top of my head, should I try it out and see?

Reply:
I just did a test on macOS, and the executable copied by `stack install` is 41 MiB.  After `strip`ing it, it goes down to 32 MiB bytes.  So it does seem to make a significant difference.

Reply:
Huh... that's surprising. OK, I'll add back the stripping on macOS.